### PR TITLE
Add time range options to influx_inspect export

### PR DESCRIPTION
**Pull request by gunnaraasen**
Adds `-start-time` and `-end-time` options to the `influx_inspect export` tool.

cc @corylanou

### CODE REVIEW

File: cmd/influx_inspect/export.go [hunk 1]
```diff
@@ -18,29 +19,54 @@ type cmdExport struct {
...
+	if endTime != "" {
+		e, err := time.Parse(time.RFC3339, endTime)
+		if err != nil {
+			return nil, err
+		}
+		end = e.UnixNano()
+	} else {
+		// set end time to max if it is not set.
+		end = 1<<63 - 1
+	}
```
Comment (corylanou) [inline cmd/influx_inspect/export.go]: Need a check somewhere to see if end time is less than start time (unless I missed it)

Comment (corylanou) [inline cmd/influx_inspect/export.go]: doh, just found it :-)

**Comment by corylanou** [comment]
When writing out this file because we are now using a time range, we should put a comment in the header of the file that states it was dumped with the specified time range.

**Comment by corylanou** [comment]
Looks good, one minor nit but not enough to stop it from shipping.  On a side note, this tool literally has no tests...  (not because of you, just in general) That tends to make for an unhappy @beckettsean at a future date (usually late at night).

👍

File: cmd/influx_inspect/export.go [hunk 2]
```diff
@@ -18,29 +19,54 @@ type cmdExport struct {
...
+	if endTime != "" {
+		e, err := time.Parse(time.RFC3339, endTime)
+		if err != nil {
+			return nil, err
+		}
+		end = e.UnixNano()
+	} else {
+		// set end time to max if it is not set.
+		end = 1<<63 - 1
```
Comment (jwilder) [inline cmd/influx_inspect/export.go]: Could we use `math.MaxInt64` here?

**Comment by jwilder** [comment]
One questions, but 👍 otherwise.

File: cmd/influx_inspect/export.go (hunk 2 above)
Comment (gunnaraasen) [inline cmd/influx_inspect/export.go]: Good point, `math.MaxInt64` is more clear. I'll update.

**Pull request merged**